File: AtlasWeb/Core/wwwroot/app/NebulaClient/StorageManager/DefinitionManagement/TTModuleDef.ts
import { Guid } from "../../Mscorlib/Guid";
import { TTAssemblyDef } from "./TTAssemblyDef";

export class TTModuleDef {
    public ModuleDefID: Guid;
    public FolderDefID: Guid;
    public Name: string;
    public Description: string;
    public Version: number;
    public AssemblyDefID: Guid;
    public AssemblyDef: TTAssemblyDef;
    public PTNodeID: Guid;
    public LastBuildDate: Date;
    public ToString(): string {
        return null;
    }
}

File: ATLASDEV/AtlasModel/Enterprise/TTModuleDef.cs
using System;

namespace AtlasModel.Enterprise
{
    public partial class TTModuleDef
    {
        public Guid ModuleDefId { get; set; }
        public string Name { get; set; }
        public Guid FolderDefId { get; set; }
        public int Version { get; set; }
        public Guid? PTNodeId { get; set; }
        public short CheckOutStatus { get; set; }
        public string CheckOutId { get; set; }
        public DateTime? LastBuildDate { get; set; }
        public string Description { get; set; }
        public Guid AssemblyDefId { get; set; }
    }
}